transport.go

```diff
@@ -430,3 +439,34 @@ func (t *Transport) maybeHandleStatelessReset(data []byte) bool {
 	}
 	return false
 }
+
+func (t *Transport) handleNonQUICPacket(p receivedPacket) {
+	// Strictly speaking, this is racy,
+	// but we only care about receiving packets at some point after ReadNonQUICPacket has been called.
+	if !t.readingNonQUICPackets.Load() {
+		return
+	}
+	select {
+	case t.nonQUICPackets <- p:
+	default:
```

Comment:
Is there some stats we could track how many packets are dropped?

Reply:
We can invoke the drop call on the tracer here, good point.

Reply:
Done.